Podfile.lock

```diff
@@ -125,7 +125,7 @@ EXTERNAL SOURCES:
     :branch: signal-master
     :git: https://github.com/WhisperSystems/JSQMessagesViewController.git
   SignalServiceKit:
-    :path: "."
```

Comment:
oh geeze. Seems like it's a ruby thing: https://github.com/CocoaPods/CocoaPods/issues/4771

We can either:
1. track a `.ruby-version` file with the project and use it.
2. you can tell me version you're running and I'll meet you there.

FYI - I'm at 2.41
```
→ ruby --version
ruby 2.4.1p111 (2017-03-22 revision 58053) [x86_64-darwin16]
```